accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/slingsettings/ExtendedSlingSettingsServiceImpl.java

```diff
@@ -49,8 +49,12 @@ public void activate(Config config) {
         if(isCloudReady) {
             extendedRunmodes.add(ADDITIONAL_RUNMODE_CLOUD);
         }
-        List<String> additionalRunmodes = Arrays.asList(config.additionalRunmodes());
-        extendedRunmodes.addAll(additionalRunmodes);
+
+        if (config.additionalRunmodes() != null) {
+            List<String> additionalRunmodes = Arrays.asList(config.additionalRunmodes());
+            extendedRunmodes.addAll(additionalRunmodes);
+        }
```

Comment:
The stacktrace shows that `config.additionalRunmodes()` returns null even though in theory, the annotation interface https://github.com/Netcentric/accesscontroltool/blob/789f7d5aab9cdfe0864b6033a68e74173745131a/accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/slingsettings/ExtendedSlingSettingsServiceImpl.java#L34 defines a default value and therefore the value should never be null.

But anyway, the stacktrace shows this can be  a problem and this PR fixes should fix it, merging it.

Reply:
It can be null in DS 1.3 in case no entry has been provided. This has been fixed in DS 1.4 (https://docs.osgi.org/specification/osgi.cmpn/7.0.0/service.component.html#d0e46608) which will now instead return an empty array